### PR TITLE
Ated 2319

### PR DESCRIPTION
**Pull request by adamtrousdale**

### CODE REVIEW

File: app/uk/gov/hmrc/agentclientmandate/services/ClientMandateCreateService.scala [hunk 1]
```diff
@@ -42,9 +42,11 @@ trait ClientMandateCreateService {
           clientMandateDto.contactDetails.phone
         )
       ),
+      clientParty = None,
       currentStatus = createPendingStatus(credId),
       statusHistory = None,
-      service = Service(None, clientMandateDto.service.name)
+      subscription = Subscription(None, service = Service(clientMandateDto.service.name.toLowerCase, clientMandateDto.service.name))
+      //service = Service(None, clientMandateDto.service.name)
```
Comment (avinash-anand) [inline app/uk/gov/hmrc/agentclientmandate/services/ClientMandateCreateService.scala]: should we remove this, if not needed?

File: app/uk/gov/hmrc/agentclientmandate/connectors/EmailConnector.scala [hunk 2]
```diff
@@ -50,7 +51,7 @@ trait EmailConnector extends ServicesConfig{
 
     http.POST(postUrl, jsonData).map { response =>
       response.status match {
-        case 202 => response
+        case OK => response
```
Comment (avinash-anand) [inline app/uk/gov/hmrc/agentclientmandate/connectors/EmailConnector.scala]: looking at code of email service, it should be Accepted(202).

File: app/uk/gov/hmrc/agentclientmandate/services/ClientMandateCreateService.scala (hunk 1 above)
Comment (adamtrousdale) [inline app/uk/gov/hmrc/agentclientmandate/services/ClientMandateCreateService.scala]: Yes please remove it

File: app/uk/gov/hmrc/agentclientmandate/connectors/EmailConnector.scala (hunk 2 above)
Comment (adamtrousdale) [inline app/uk/gov/hmrc/agentclientmandate/connectors/EmailConnector.scala]: Yep, my mistake! Should be accepted

**Pull request merged**